man/corosync.conf.5

```diff
@@ -572,6 +572,15 @@ Allow UDPU and KNET to drop packets from IP addresses that are not known
 (nodes which don't exist in the nodelist) to corosync.
 Value is yes or no.
 
+This feature is mainly to protect joining a nodes with outdated configuration
+after the cluster split. Another use case is to allow atomic merge of two
+independent clusters.
+
```

Comment:
A slight phrasing change makes it clearer I think:

"This feature is mainly to protect against the joining of nodes with outdated configurations
after a cluster split. Another use case is to allow the atomic merge of two independent clusters"